src/schema/local_discovery/city.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  GraphQLFloat,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+
+import Show from "schema/show"
+import PartnerShowSorts from "schema/sorts/partner_show_sorts"
+import Fair from "schema/fair"
+import FairSorts from "schema/sorts/fair_sorts"
+import EventStatus from "schema/input_fields/event_status"
+
+import cityData from "./city_data.json"
+
+const LOCAL_DISCOVERY_RADIUS_KM = 75
+
+const LatLngType = new GraphQLObjectType({
+  name: "LatLng",
+  fields: {
+    lat: {
+      type: GraphQLFloat,
+      resolve: obj => {
+        return obj.lat
+      },
+    },
+    lng: {
+      type: GraphQLFloat,
+      resolve: obj => {
+        return obj.lng
+      },
+    },
```

Comment:
Once you have such an object, its fields will probably never be null?

Reply:
graphql-js has a default field resolver that will try to use the name of the field on the data being passed to its parent, so these don’t need a resolver at all.

Reply:
Is that level of magic allowed outside of Rails :trollface: ?

Reply:
Magic is the best